### DISCUSSION TITLE
GoFa support

### DISCUSSION THREAD

**Discussion by edurnemorales** [visose/Robots#288] [Q&A] [answered]
Hello Robots, 
I read in the release notes of version 1.8.0 that you added the support for ABB GoFa. In which library could I find it?
Also, is there a database of the robots included in each library?
Thank you in advance

**Comment by visose** [comment, marked as answer]
Hi Edurne,  you can find it in the `Swoosh` library.

There's no database, but if you clone the repo and switch to the `libraries` branch,  you can use the search function of a text editor to look through the content of all the xmls in the folder.

**Reply by edurnemorales** [reply]
Got there! Thank you
